lib/upgrade.js

```diff
@@ -2,9 +2,12 @@ const logger = require('@blackbaud/skyux-logger');
 
 const appDependencies = require('../lib/app-dependencies');
 const jsonUtils = require('../lib/utils/json-utils');
-const cleanup = require('../lib/cleanup');
+const npmInstall = require('./utils/npm-install');
 
-async function upgrade() {
+async function upgrade(argv) {
+
+  // Did the consumer pass `--no-install` via the command line?
+  const runInstall = (argv.install !== false);
```

Comment:
By default, running `skyux upgrade` will update the consumer's package.json file with the latest packages and then run `npm install` as a convenience. If the consumer wishes to skip the installation process, they can append `--no-install` to the upgrade command.